src/components/App/App.js

```diff
@@ -1,12 +1,22 @@
+import React, { } from 'react';
 import Header from '../Header/Header';
+import Footer from '../Footer/Footer';
+import LanguagesSupported from '../LanguagesSupported/LanguagesSupported';
 import './App.css';
+import { Route } from "react-router-dom";
 
-function App() {
+  
+const App = () => {
   return (
     <div className="App">
-      <header className="App-header">
-        <Header />
-      </header>
+      <Header />
+      <Route
+        path="/languages"
+        render={() => {
```

Comment:
Per the docs I believe it would be cleaner to do:
```
<Route path="/languages">
  <LanguagesSupported />
</Route>
```
Render is typically used only if you need to add logic.

Reply:
Thank you!